screens/auth/SignUpScreen.js

```diff
@@ -158,6 +159,13 @@ export default class SignUpScreen extends React.Component {
       const customerId = await this.addCustomer();
       this._clearState();
       await this._asyncSignUp(customerId);
+      // register this user in the Sentry logger
+      Sentry.configureScope((scope) => {
+        scope.setUser({
+          id: customerId,
+          phoneNumber: formattedPhoneNumber,
+        });
+      });
     } catch (err) {
       console.error('[SignUpScreen] (handleSubmit) Airtable:', err);
```

Comment:
sentry sdk automatically captures unhandled promises and exceptions -- but we are want to capture and send the errors that are already handled/caught by the app, which is why we have to explicitly call `Sentry.captureException`

Wilson's PR for reference for javascript examples: https://github.com/calblueprint/unloop/pull/73/files

We should be logging when:
1. sections of the codebase are catching errors through `.catch(error)` (i.e airtable queries)
2. sections of the app where there is an error state being logged
- perfect candidate of #2 is the `customers.length >1` case in the `LoginScreen` where it's a database error that isn't an uncaught promise or exception but we need to send

Example of adding extra information for context in capturing the exception
```
Sentry.configureScope(function(scope) {
          scope.setExtra('screen', 'SignUpScreen');
          scope.setExtra('action', 'handleSubmit');
        });
        Sentry.captureException(err);
```